script/bootstrap.py

```diff
@@ -110,6 +110,9 @@ def parse_args():
                       help='The static library path of libchromiumcontent.')
   parser.add_argument('--defines', default='',
                       help='The build variables passed to gyp')
+  parser.add_argument('--sccache',
```

Comment:
I would rename it to `cc_wrapper`.
Because technically it can be any binary, not only `sccache`. And it would also match GN's arg name.